Align command-line options for clickhouse-format with clickhouse-client

rishabh1815769 opened the pull request:
<!---
A technical comment, you are free to remove or leave it as it is when PR is created
The following categories are used in the next scripts, update them accordingly
utils/changelog/changelog.py
tests/ci/cancel_and_rerun_workflow_lambda/app.py
-->
### Changelog category (leave one):
- Improvement


### Changelog entry (a [user-readable short description](https://github.com/ClickHouse/ClickHouse/blob/master/docs/changelog_entry_guidelines.md) of the changes that goes into CHANGELOG.md):

- Updated `clickhouse-format` to accept `--highlight` as an alias for `--hilite`.
- Updated `clickhouse-client` to accept `--hilite` as an alias for `--highlight`.
- Updated `clickhouse-format`  documentation to reflect the change.

### Documentation entry for user-facing changes

- [x] Documentation is written (mandatory for new features)

<!---
Directly edit documentation source files in the "docs" folder with the same pull-request as code changes

or

Add a user-readable short description of the changes that should be added to docs.clickhouse.com below.

At a minimum, the following information should be added (but add more as needed).
- Motivation: Why is this function, table engine, etc. useful to ClickHouse users?

- Parameters: If the feature being added takes arguments, options or is influenced by settings, please list them below with a brief explanation.

- Example use: A query or command.
-->

Addresses #83686

rishabh1815769 committed:
Align command-line options for clickhouse-format with clickhouse-client

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/ClickHouse/ClickHouse?pullRequest=84806) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/ClickHouse/ClickHouse?pullRequest=84806) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/ClickHouse/ClickHouse?pullRequest=84806) it.</sub>

alexey-milovidov reviewed (commented):
Tests needed. Look at `tests/queries/0_stateless`, `.sh`

rishabh1815769 committed:
Merge branch 'master' of github.com:ClickHouse/ClickHouse into inconsistent-cli-options-highlight-hilite

rishabh1815769 committed:
Test support for --highlight argument in clickhouse-format

rishabh1815769 committed:
Merge branch 'master' of github.com:ClickHouse/ClickHouse into inconsistent-cli-options-highlight-hilite

rishabh1815769 committed:
Added more tests for clickhouse-format

rishabh1815769 commented:
Hi @alexey-milovidov, I’ve added tests for clickhouse-format, but I’m not sure how to write tests for clickhouse-client, as the --highlight option only works in interactive mode. Could you please review the tests I’ve added and provide guidance on how to approach testing for the clickhouse-client? Thank you!

rishabh1815769 committed:
Merge branch 'master' of github.com:ClickHouse/ClickHouse into inconsistent-cli-options-highlight-hilite

rishabh1815769 committed:
Added expect test to validate if --hilite is working as expected in clickhouse-client

rishabh1815769 commented:
Closes #83686

rishabh1815769 committed:
Made expect test file executable

rishabh1815769 commented:
Hi @alexey-milovidov, kindly review the PR.

alexey-milovidov reviewed (approved):
Nice.
